.github/workflows/ci.yml

```diff
@@ -0,0 +1,175 @@
+          - pg_major: 10
+            edb_version: '10.9-2'
+            postgis_version: '2.5.3'
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v1
+
+      - name: Start PostgreSQL ${{ matrix.pg_major }} (docker)
+        if: startsWith(matrix.os, 'ubuntu')
+        env:
+          IMAGE: npgsql/postgres:${{ matrix.pg_major }}
+        run: docker run --detach --name postgresql -p 5432:5432 $IMAGE
+        shell: bash
+
+      - name: Start PostgreSQL ${{ matrix.pg_major }} (install)
+        if: startsWith(matrix.os, 'windows')
+        env:
+          PG_MAJOR: ${{ matrix.pg_major }}
+          EDB_VERSION: ${{ matrix.edb_version }}
+          POSTGIS_VERSION: ${{ matrix.postgis_version }}
+        run: |
+          # Install PostgreSQL
+          curl -o pgsql.zip -L https://get.enterprisedb.com/postgresql/postgresql-${EDB_VERSION}-windows-x64-binaries.zip
+          unzip pgsql.zip -x 'pgsql/include/**' 'pgsql/doc/**' 'pgsql/pgAdmin 4/**' 'pgsql/StackBuilder/**'
+
+          # Match Npgsql CI Docker image and stash one level up
+          cp {$GITHUB_WORKSPACE/.build/docker,pgsql}/server.crt
+          cp {$GITHUB_WORKSPACE/.build/docker,pgsql}/server.key
+
+          # Install PostGIS
+          POSTGIS_FILE="postgis-bundle-pg${PG_MAJOR}-${POSTGIS_VERSION}x64"
+          curl -o postgis.zip -L https://download.osgeo.org/postgis/windows/pg${PG_MAJOR}/${POSTGIS_FILE}.zip
+          unzip postgis.zip -d postgis
+          cp -a postgis/$POSTGIS_FILE/. pgsql/
+
+          # Start PostgreSQL
+          pgsql/bin/initdb -D pgsql/PGDATA -E UTF8 -U postgres
+          pgsql/bin/pg_ctl -D pgsql/PGDATA -l logfile -o '-c max_prepared_transactions=10 -c ssl=true -c ssl_cert_file=../server.crt -c ssl_key_file=../server.key' start
+
+          # Configure test account
+          pgsql/bin/psql -U postgres -c "CREATE ROLE npgsql_tests SUPERUSER LOGIN PASSWORD 'npgsql_tests'"
+          pgsql/bin/psql -U postgres -c "CREATE DATABASE npgsql_tests OWNER npgsql_tests"
+        shell: bash
+
+      - name: Setup .NET Core SDK
+        uses: actions/setup-dotnet@v1.0.2 # Issues with DOTNET_ROOT
+        with:
+          dotnet-version: ${{ matrix.dotnet }}
+
+      - name: Test (EFCore.PG.Tests)
+        run: dotnet test test/EFCore.PG.Tests --configuration Debug
+        shell: bash
+
+      - name: Test (EFCore.PG.FunctionalTests)
+        run: dotnet test test/EFCore.PG.FunctionalTests --configuration Debug
+        shell: bash
+
+      - name: Test (EFCore.PG.Plugins.FunctionalTests)
+        run: dotnet test test/EFCore.PG.Plugins.FunctionalTests --configuration Debug
+        shell: bash
+
+      - name: Pack NuGet packages (CI versions)
+        env:
+          GITHUB_SHA: ${{ github.sha }}
+        run: |
+          VERSION_SUFFIX="ci.$(date -u +%Y%m%dT%H%M%S)+sha.${GITHUB_SHA:0:9}"
+          dotnet pack src/EFCore.PG          --configuration Release --output nupkgs/ci --version-suffix $VERSION_SUFFIX
```

Comment:
Unless I'm mistaken, `--include-symbols` causes both packages to be created, so no need to invoke pack twice. Or is this different with snupkg somehow?

Reply:
Actually with snupkg I don't think `--include-symbols` even does anything...

Reply:
Nice catch! Just ran it locally and no need for the extra noise:
```
PS ~\rider\efcore.pg> dotnet pack src/EFCore.PG -o temp
Microsoft (R) Build Engine version 16.3.0+0f4c62fea for .NET Core
Copyright (C) Microsoft Corporation. All rights reserved.

  Restore completed in 28.13 ms for ~\rider\efcore.pg\src\EFCore.PG\EFCore.PG.csproj.
  EFCore.PG -> ~\rider\efcore.pg\src\EFCore.PG\bin\Debug\netstandard2.1\Npgsql.EntityFrameworkCore.PostgreSQL.dll
  Successfully created package '~\rider\efcore.pg\temp\Npgsql.EntityFrameworkCore.PostgreSQL.3.1.0.nupkg'.
  Successfully created package '~\rider\efcore.pg\temp\Npgsql.EntityFrameworkCore.PostgreSQL.3.1.0.snupkg'.
```